homeassistant/components/template/template_entity.py

```diff
@@ -100,6 +101,13 @@
 ).extend(TEMPLATE_ENTITY_AVAILABILITY_SCHEMA.schema)
 
 
+TEMPLATE_ENTITY_OPTIMISTIC_SCHEMA = vol.Schema(
+    {
+        vol.Optional(CONF_OPTIMISTIC, default=False): cv.boolean,
+    }
+)
```

Comment:
If we don't ever use this as a voluptuous schema, just make it a dict:

```suggestion
TEMPLATE_ENTITY_OPTIMISTIC_SCHEMA = {
    vol.Optional(CONF_OPTIMISTIC, default=False): cv.boolean,
}
```

The same probably goes for some of the other base schemas.